### PR TITLE
Sort files by time of modification 

### LINKED ISSUE #17: Add a flag to sort the files by the time of modification

### LINKED ISSUE #28: Adding a flag to group files and directories while sorting them by the time of modification

### PR DESCRIPTION
**Pull request by Aeswolf**

### CODE REVIEW

File: ls/ls.go [hunk 1]
```diff
@@ -141,3 +152,100 @@ func (l *LS) showDirStructure() error {
...
+		month := fileStat.ModTime().UTC().Month()
+		day := fileStat.ModTime().UTC().Day()
+		hour := fileStat.ModTime().UTC().Hour()
+		minute := fileStat.ModTime().UTC().Minute()
+		second := fileStat.ModTime().UTC().Second()
+		timeFormat := formatTime(hour, minute, second)
+		dateFormat := formatDate(year, int(month), day)
+
+		displayFormat := fmt.Sprintf("%s Date: %s Time: %s", fileName, dateFormat, timeFormat)
+
```
Comment (Bostigger) [inline ls/ls.go:222]: should we remove the empty space here

Comment (Bostigger) [inline ls/ls.go]: should we remove the empty space, 
**_before_**
displayFormat := fmt.Sprintf("%s Date: %s Time: %s", fileName, dateFormat, timeFormat)

fmt.Fprintln(l.StdOut, displayFormat)

**_suggested_** 
displayFormat := fmt.Sprintf("%s Date: %s Time: %s", fileName, dateFormat, timeFormat)
fmt.Fprintln(l.StdOut, displayFormat)

**Review by Bostigger — APPROVED** [review]
Good job!

File: ls/ls.go (hunk 1 above)
Comment (Aeswolf) [reply ls/ls.go]: Sure why not. Thanks

**Review by profclems — APPROVED** [review]
LGTM!

**Pull request merged**